example/CMakeLists.txt

```diff
@@ -1,35 +1,8 @@
-cmake_minimum_required(VERSION 3.13...3.24)
-project(p4estExamples LANGUAGES C)
-
-include(CheckSymbolExists)
-include(CheckIncludeFile)
-
 option(mpi "use MPI" off)
```

Comment:
you should probably remove this line: `mpi` option is already defined at top-level in `cmake/options.cmake`; it was fine when examples were in separate cmake project, but not anymore here.

Reply:
Thanks for your feedback. The option `mpi` was redundant and the building works fine without it.